src/templates/users/_edit.twig

```diff
@@ -89,7 +89,7 @@
                     name: 'email',
                     value: user.email,
                     type: 'email',
-                    errors: user.getErrors('email')|merge(user.getErrors('unverifiedEmail')),
```

Comment:
We already copy/clear `unverifiedEmail` errors to `email` [in the controller](https://github.com/craftcms/cms/blob/develop/src/controllers/UsersController.php#L1444-L1448), so we shouldn't need to render both here.

If we do, we can end up rendering the same error twice, in the case of an invalid email.